initial commit

builtbyteran opened the pull request.

Commit:
initial commit

ronyrosenberg commented on main.js:
```diff
@@ -0,0 +1,48 @@
+    return "---" + peg.join(" ");
+  });
+
+  currentBoard.map(function (peg) {
```
no need for map, use forEach

ronyrosenberg commented on main.js:
```diff
@@ -0,0 +1,48 @@
+  return;
+}
+
+const moveDisc = function (sourcePeg, targetPeg) {
```
Add a check before originPeg.pop() to ensure originPeg.length > 0

builtbyteran committed:
Added reset button and logic

builtbyteran committed:
Refactored moveDisc, added comments

builtbyteran committed:
Added markup and styling

builtbyteran committed:
updated readme

builtbyteran committed:
Installed wesbos config, switched to arrow functions

builtbyteran committed:
Refactor Towers of Hanoi to TypeScript

builtbyteran committed:
Added UI for user to movedisc

builtbyteran committed:
Added bootsrap, implemented new features

builtbyteran committed:
Refactor: Add visual UI and TypeScript support

- Convert console-based game to interactive web interface
- Add Bootstrap styling and responsive layout
- Implement dropdown menus for peg selection
- Add dynamic disk count selection (3-5 disks)
- Update game rendering with visual peg/disk display

builtbyteran committed:
Fixed readme

builtbyteran committed:
Deployed and updated readme

Commit:
Fixed NaN error on disks select

builtbyteran committed:
Updated instructions post deploy to reflect new changes made in js file

builtbyteran committed:
Reordered style links

builtbyteran committed:
Updated readme

Commit:
Updated comments, added wesbos typescript config

Commit:
Updated instructions in UI

Commit:
Disabled selected options

Commit:
Styled the disks

Commit:
Updated readme
